site/styles/_variables.scss

```diff
@@ -89,7 +87,8 @@ $signin-text-color: $body-color !default;
 $signin-text-color-hover: $body-color !default;
 $signin-border-color: transparent !default;
 $signin-border-color-hover: transparent !default;
-$signin-background: transparent !default;
+$signin-background: rgba($color: $body-color, $alpha: 0.06) !default;
```

Comment:
That sure is the tiniest alpha - does it work across light/dark themes?

Reply:
Yes. It's a background colour so the contrast doesn't need to be very high to be noticable.